modules/repository-s3/src/internalClusterTest/java/org/elasticsearch/repositories/s3/S3BlobStoreRepositoryTests.java

```diff
@@ -323,6 +393,10 @@ public void maybeTrack(final String request, Headers requestHeaders) {
                 trackRequest("PutMultipartObject");
             } else if (Regex.simpleMatch("PUT /*/*", request)) {
                 trackRequest("PutObject");
+            } else if (Regex.simpleMatch("POST /*/?delete", request)) {
+                trackRequest("DeleteObjects");
+            } else if (Regex.simpleMatch("DELETE /*/*?*uploadId=*", request)) {
```

Comment:
Is the `*` before the `uploadId` necessary? On the [s3 API page](https://docs.aws.amazon.com/AmazonS3/latest/API/API_AbortMultipartUpload.html), I see it directly as `?uploadId=`.

Reply:
it's possible that there are multiple query params, or it might be in the future

Reply:
I removed the `*`. We can adjust it again if it fails in future.